Firmware updates and more info for telemetry id 4198

Revilo91 opened the pull request:
I found some changes in dashboard api endpoint in the newest (1.5.5) firmware.

And new info for telemetry id 4198

Revilo91 committed:
Update ComfoClimeAPI.md

4198 = Heatpump percent?

Revilo91 committed:
Firmware update 1.5.5 changes

helgeklein commented:
@Revilo91 I've also tested the dashboard API with the new firmware 1.5.5 and would like to contribute some changes to this PR. I hope you don't mind if I create a PR in your fork.

helgeklein committed:
Updated the dashboard API endpoint field description.

- Added a "Min. firmware" column
- Added description of new fields in 1.5.5
- Mentioned which field are not always available
- Merged two JSON examples back into one

helgeklein commented:
@Revilo91 Here's [my PR](https://github.com/Revilo91/comfoclime_api/pull/1). If you accept it, the changes should appear here in this PR, too.
(CC @msfuture)

Revilo91 committed:
Merge pull request #1 from helgeklein/main

Updated the dashboard API endpoint field description.

Revilo91 commented:
@helgeklein your [PR](https://github.com/Revilo91/comfoclime_api/pull/1) is merged 👍
